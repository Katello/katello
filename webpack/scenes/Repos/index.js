import React from 'react';

export default () => <h1>Repos Page</h1>;
