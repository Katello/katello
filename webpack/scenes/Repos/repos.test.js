import React from 'react';
import { configure, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Repos from './index';

configure({ adapter: new Adapter() });

describe('Nav component', () => {
  it('renders the navigation', () => {
    const wrapper = mount(<Repos />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
