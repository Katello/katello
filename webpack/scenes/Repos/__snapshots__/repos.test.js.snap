// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component renders the navigation 1`] = `
<Component>
  <h1>
    Repos Page
  </h1>
</Component>
`;
